src/platform/site-wide/announcements/config/index.js

```diff
@@ -32,6 +32,7 @@ const config = {
       name: 'veterans-day-proclamation-edited',
       paths: /^\/$/,
       component: VeteransDayProclamation,
+      expiresAt: '2019-11-12',
```

Comment:
This will use the user's timezone. I think this makes sense.

Reply:
:thinking: Yeah, if the timezone is messed up on their machine, I think they'll either see an alert when they shouldn't, or not see an alert when they should. I think it's not worth a more complicated implementation, though. :+1: